README.md

```diff
@@ -15,27 +27,58 @@ Variables without a default value and not marked optional *MUST* be defined for
 | LAB_RESULTS_MESH_OUTBOUND_QUEUE_NAME | lab_results_mesh_outbound | The name of the outbound (to MESH) message queue
 | LAB_RESULTS_MESH_INBOUND_QUEUE_NAME  | lab_results_mesh_inbound  | The name of the inbound (from MESH) message queue
 | LAB_RESULTS_GP_OUTBOUND_QUEUE_NAME   | lab_results_gp_outbound   | The name of the outbound (to GP System) message queue
+| LAB_RESULTS_AMQP_BROKERS             | amqp://localhost:5672     | A comma-separated list of URLs to AMQP brokers (*)
+| LAB_RESULTS_AMQP_USERNAME            |                           | (Optional) username for the AMQP server
+| LAB_RESULTS_AMQP_PASSWORD            |                           | (Optional) password for the AMQP server
+| LAB_RESULTS_AMQP_MAX_REDELIVERIES    | 3                         | The number of times an message will be retried to be delivered to consumer. After exhausting all retires, it will be put on DLQ.<queue_name> dead letter queue
```

Comment:
Is this actually retries, or total attempts? I.e., does this mean that the maximum number of attempts is 3, or 4, by default?
```suggestion
| LAB_RESULTS_AMQP_MAX_REDELIVERIES    | 3                         | The number of times an message will be retried to be delivered to consumer. After exhausting all retries, it will be put on DLQ.<queue_name> dead letter queue
```

Reply:
We've set `LAB_RESULTS_AMQP_MAX_REDELIVERIES` to `3` in the `application.yml` and `docker-compose.yml`.

> jms.redeliveryPolicy.maxRedeliveries controls when an incoming message is rejected based on the number of times it has been redelivered, the default value is (-1) disabled. A value of zero would indicate no message redeliveries are accepted, a value of five would allow a message to be redelivered five times, etc.

https://qpid.apache.org/releases/qpid-jms-0.56.0/docs/index.html

We could re-word this?

Reply:
So explicitly retries. I've seen confusion about the distinction between attempts and retries in other projects. We could just add similar comments about -1 and 0.

Also just noticed "an message"

Reply:
I've added a comment mentioning the default JMS value